index.d.ts

```diff
@@ -24,14 +25,16 @@ declare namespace meow {
 		- `type`: Type of value. (Possible values: `string` `boolean` `number`)
 		- `alias`: Usually used to define a short flag alias.
 		- `default`: Default value when the flag is not specified.
+		- `multiple`: Indicates a flag can be set multiple times. Returns an array. (Default: false)
```

Comment:
*What* returns an array? It's not clear enough.

Reply:
Valid point. Maybe drop the word `return` altogether?

How about `Values are turned into an array (in order?)`?

Reply:
Went with the shorter form